curriculum/challenges/chinese/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md

```diff
@@ -1,24 +1,24 @@
 ---
 id: 5a8b073d06fa14fcfde687aa
-title: Exercise Tracker
+title: Exercise 追踪器
 challengeType: 4
 forumTopicId: 301505
 dashedName: exercise-tracker
 ---
 
 # --description--
 
-Build a full stack JavaScript app that is functionally similar to this: <https://exercise-tracker.freecodecamp.rocks/>. Working on this project will involve you writing your code using one of the following methods:
+构建一个 JavaScript 的全栈应用，在功能上与这个应用相似： <https://exercise-tracker.freecodecamp.rocks/>。 可以采用下面的一种方式完成这个挑战：
 
--   Clone [this GitHub repo](https://github.com/freeCodeCamp/boilerplate-project-exercisetracker/) and complete your project locally.
--   Use [our repl.it starter project](https://repl.it/github/freeCodeCamp/boilerplate-project-exercisetracker) to complete your project.
--   Use a site builder of your choice to complete the project. Be sure to incorporate all the files from our GitHub repo.
+-   克隆 [this GitHub repo](https://github.com/freeCodeCamp/boilerplate-project-exercisetracker/) 并在本地完成项目。
+-   使用 [repl.it 初始化项目](https://repl.it/github/freeCodeCamp/boilerplate-project-exercisetracker) 来完成项目。
```

Comment:
Why is one link text translated but the other isn't?

Reply:
I tagged the string on Crowdin.